packages/config-cli/README.md

```diff
@@ -16,3 +16,11 @@ Dump the screenshots from different host and tag
 ./bin/bmc.js screenshot --host HOST --tag PR-TAG
 
 ```
+
+## Usage -- Screenshots-server
+
+Create a temporary server from a config bundle file and dump the screenshots
+
+```bash
+./bin/bmc.js screenshot-server - config s3://..../config.json.gz
```

Comment:
I think this should just be a argument to `screenshot`

eg 

```
./bin/bmc.js screenshot --config s3://.../config.json.gz
```

Reply:
Might be good to separate to two different cli? I think this will make the cli logic not too complex.

Reply:
should be pretty simiple logic ?

something like 
```
if (this.config.value) 
 const host = await startServer()
 return await screenshot(host);
}

return await screenshot(this.host.value)
```

they are both doing the same thing, screenshotting some host, so to me they are the same command